pages/carouselVenueElbdeckEventLocation.jsx

```diff
@@ -0,0 +1,124 @@
+import Head from "next/head";
+import Image from "next/image";
+import styled from "styled-components";
+import arrowLeft from "../public/icons/arrow-left.svg";
+import Carousel from "react-multi-carousel";
+import "react-multi-carousel/lib/styles.css";
+import carouselVenue1Pic1 from "../public/images/carouselVenue1Pic1.jpeg";
+import carouselVenue1Pic2 from "../public/images/carouselVenue1Pic2.jpeg";
+import carouselVenue1Pic3 from "../public/images/carouselVenue1Pic3.jpeg";
+import carouselVenue1Pic4 from "../public/images/carouselVenue1Pic4.jpeg";
+
+export default function CarouselVenueElbdeckEventLocation() {
+  const responsive = {
+    superLargeDesktop: {
+      breakpoint: { max: 4000, min: 3000 },
+      items: 5,
+    },
+    desktop: {
+      breakpoint: { max: 3000, min: 1024 },
+      items: 3,
+    },
+    tablet: {
+      breakpoint: { max: 1024, min: 464 },
+      items: 2,
+    },
+    mobile: {
+      breakpoint: { max: 464, min: 0 },
+      items: 1,
+    },
+  };
+  return (
+    <>
+      <Head>
+        {" "}
+        <title>Dream Day Wedding Planner</title>
+      </Head>
+      <PufferPageTop />
+      <LinkStyle href="/galleryVenue">
+        <Image src={arrowLeft} alt="back" width={30} height={30} />
+      </LinkStyle>
+      <HeadingStyle>Photo Gallery</HeadingStyle>
+      <GalleryTitle>ELBDECK EVENT LOCATION</GalleryTitle>
+      <Carousel responsive={responsive}>
+        <div>
+          <Image
+            src={carouselVenue1Pic1}
+            alt="venue picture"
+            width={400}
+            height={250}
+          />
+        </div>
+        <div>
+          <Image
+            src={carouselVenue1Pic2}
+            alt="venue picture"
+            width={400}
+            height={250}
+          />
+        </div>
+        <div>
+          <Image
+            src={carouselVenue1Pic3}
+            alt="venue picture"
+            width={400}
+            height={250}
+          />
+        </div>
+        <div>
+          <Image
+            src={carouselVenue1Pic4}
+            alt="venue picture"
+            width={400}
+            height={250}
+          />
+        </div>
+      </Carousel>
```

Comment:
Once you have dynamic data there would be no need to hardcode this. Instead you could simply `.map()` of the image array of the current venue.